static/scss/top-app-bar.scss

```diff
@@ -59,52 +65,163 @@
       }
 
       &.button {
-        background: $primary;
+        @include media-breakpoint-up(md) {
+          background: $primary;
+          color: white;
+          border-radius: 3px;
+          box-shadow: inset 0 0 2px 0 rgba(255, 255, 255, 0.57);
+
+          &:hover {
+            background: black;
+          }
+        }
+      }
+    }
+  }
+
+  @include media-breakpoint-down(sm) {
+    display: none;
+    position: absolute;
+    padding-top: 70px;
+    background: rgba(0, 0, 0, 0.91);
+    left: 0;
+    right: 0;
+    bottom: 0;
+    top: 70px;
+    height: auto !important;
+    z-index: 999;
+
+    li {
+      display: block;
+      padding: 10px 0;
+      margin: 0 auto;
+      max-width: 170px;
+      text-align: left;
+
+      a {
         color: white;
+        font-size: 18px;
+        font-weight: 600;
+        line-height: 25px;
+        padding: 0;
+        margin-top: 5px;
 
         &:hover {
-          background: black;
+          color: white;
         }
       }
     }
   }
 }
 
+.authenticated-menu {
+  display: none !important;
+
+  @include media-breakpoint-down(sm) {
+    display: block !important;
+  }
+}
+
 .navbar-collapse.collapsing,
 .navbar-collapse.show {
   display: flex;
+  @include media-breakpoint-down(sm) {
+    display: block;
+  }
 }
 
 .site-logo {
-  width: 154px;
-  height: 47.5px;
+  width: 65px;
+  height: 35px;
+  @include media-breakpoint-down(sm) {
+    width: 46px;
+    height: 25px;
+  }
 }
 
-.navbar-toggler.nav-opener {
-  border: 1px solid $primary;
-  border-radius: 2px;
-  color: $primary;
-  margin: 5px 0 0;
-  padding: 8px 6px 6px;
+.navbar-brand {
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  padding-top: 0;
+
+  .mitx-online-link {
+    font-size: 24px;
+    color: #000000;
+    font-weight: bold;
+    line-height: 24px;
+    letter-spacing: 0;
+    padding-left: 15px;
```

Comment:
Can we change the margin definition for `.divider-large` to `0 15px` so we have the spacing defined in one place instead of adding padding to the link here?